packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -294,16 +294,15 @@ export const guides = [
             guideName: "configure-signon-policy"
           },
           {
-            title: "Embedded authentication",
+            title: "Advanced use cases",
             subLinks: [
               {
-                title: "Email use cases",
-                subLinks: [
-                  {
-                    title: "Custom password recovery",
-                    guideName: "oie-embedded-sdk-use-case-custom-pwd-recovery-mfa"
-                  }
-                ]
+                title: "Custom password recovery",
+                guideName: "oie-embedded-sdk-use-case-custom-pwd-recovery-mfa"
+              },
+              {
+                title: "Device Context",
```

Comment:
Same comment as above. Device Context doesn't seem like a good title for this guide.